src/main/java/de/tum/in/www1/artemis/domain/TextAssessmentKnowledge.java

```diff
@@ -0,0 +1,37 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.persistence.*;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@Entity
+@Table(name = "text_assessment_knowledge")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class TextAssessmentKnowledge extends DomainObject {
```

Comment:
I think you could add a description / example (JavaDoc) here, because the name itself is somewhat abstract.